sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -131,6 +134,19 @@ public HttpMessage CreateMessage(PipelineOptions? options, ResponseClassifier? c
             return message;
         }
 
+        /// <summary>
+        /// TBD.
+        /// </summary>
+        /// <param name="options"></param>
+        /// <param name="classifier"></param>
+        /// <returns></returns>
+        /// <exception cref="NotImplementedException"></exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public override RestMessage CreateRestMessage(PipelineOptions options, ResponseErrorClassifier classifier)
```

Comment:
It is uncool that this shows up on the public API surface of HttpPipeline, but I wanted to avoid an overload that changes the return type per the FDG.  We can probably avoid this with @KrzysztofCwalina's pipeline prototype moving forward.